.gitignore

```diff
@@ -4,3 +4,4 @@ node_modules
 .idea
 .DS_Store
 .vscode
+kingdom_keys.js
```

Comment:
`keys.js`?

Reply:
I removed this completely since the linter looks for key.js and throws an error if it doesn't find it.  I'm keeping the file but removing the contents.  I'm hoping there's a better way though.